src/main/java/com/github/manolo8/darkbot/utils/LogUtils.java

```diff
@@ -20,16 +24,27 @@ public class LogUtils {
     public static final Path LOG_FOLDER = Paths.get("logs");
     public static final String START_TIME = LocalDateTime.now().format(FILENAME_DATE);
 
-    public static void setOutputToFile() {
+    public static void setOutputToFileAndConsole() {
         if (!Files.exists(LOG_FOLDER)) createFolder();
         else removeOld();
 
         try {
-            PrintStream output = getLogger();
-            System.setOut(output);
-            System.setErr(output);
+            OutputStream consoleOut = new FileOutputStream(FileDescriptor.out);
+            OutputStream consoleErr = new FileOutputStream(FileDescriptor.err);
+
+            OutputStream file = new FileOutputStream("logs/" + START_TIME + ".log");
+
+            OutputStream multiOut = new MultiOutputStream(consoleOut, file);
+            OutputStream bufferedOut = new BufferedOutputStream(multiOut, 128);
+            PrintStream printStreamOut = new PrintStreamWithDate(bufferedOut);
```

Comment:
This set of 3 lines is repeated, as well as the consoleOut/consoleErr, i'd put them inside a function, so that we can simplify this method to:
```java
OutputStream fileLogger = new FileOutputStream("logs/" + START_TIME + ".log");

System.setOut(createPrintStream(FileDescriptor.out, file));
System.setErr(createPrintStream(FileDescriptor.err, file));
```